pkg/gardenlet/controller/seed/seed_control.go

```diff
@@ -253,9 +253,8 @@ func (c *defaultControl) ReconcileSeed(obj *gardencorev1beta1.Seed, key string)
 		if len(associatedShoots) == 0 && len(associatedBackupBuckets) == 0 {
 			seedLogger.Info("No Shoots or BackupBuckets are referencing the Seed. Deletion accepted.")
 
-			// Debootstrap the Seed cluster.
-			if err := seedpkg.DebootstrapCluster(ctx, seedClient); err != nil {
-				message := fmt.Sprintf("Failed to unbootstrap Seed Cluster (%s).", err.Error())
+			if err := seedpkg.RunDeleteSeedFlow(seedClient, seedLogger); err != nil {
```

Comment:
Hm, why was this renamed? I would expect to either call this `DebootstrapCluster` or rename `BootstrapCluster` to `RunReconcileSeedFlow`.

Reply:
I would suggest to follow the naming of the ShootFlow as you proposed and go with BootstrapCluster -> RunReconcileSeedFlow